cypress/integration/errorPagePersian.js

```diff
@@ -1,5 +1,5 @@
 import config from '../support/config';
-import describeForLocalOnly from '../support/describeForLocalOnly';
+import describeForLocalAndTest from '../support/limitEnvRuns';
```

Comment:
This wont work as Persian error pages are [broken on test](https://www.test.bbc.com/persian/articles/cyddjz5058eo), also you would need to update the variable use on line 11 anyway